src/main/java/edu/indiana/dlib/amppd/controller/JobController.java

```diff
@@ -44,10 +45,10 @@
 	 * @param workflowId the ID of the workflow 
 	 * @param primaryfileId the ID of the primaryfile
 	 * @param parameters the parameters to use for the steps in the workflow as a map {stepId: {paramName; paramValue}}
-	 * @return outputs of the invocation returned by Galaxy
+	 * @return WorkflowOutputResult containing detailed information for the file submitted
 	 */
```

Comment:
"for the file submitted": I'd suggest "for the workflow submission on the primaryfile"